<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>979</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>810</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>810</y>
      <width>187</width>
      <height>57</height>
     </rect>
    </property>
    <property name="text">
     <string>cancel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditProjectname">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>911</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>enter your project name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enterPNButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>90</y>
      <width>187</width>
      <height>57</height>
     </rect>
    </property>
    <property name="text">
     <string>enter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>select type of the project</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>901</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="enterSP">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>310</y>
      <width>187</width>
      <height>57</height>
     </rect>
    </property>
    <property name="text">
     <string>enter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>114</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>nextButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label</zorder>
   <zorder>lineEditProjectname</zorder>
   <zorder>enterPNButton</zorder>
   <zorder>label_2</zorder>
   <zorder>lineEdit</zorder>
   <zorder>enterSP</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1218</width>
     <height>47</height>
    </rect>
   </property>
   <widget class="QMenu" name="menupage_1">
    <property name="title">
     <string>page_1</string>
    </property>
   </widget>
   <addaction name="menupage_1"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
